Implementação da HttpApi no frontend

joaojvds opened the pull request:
--Agora o wihtdraw é feito usando a httpApi e não mais o socket

joaojvds committed:
Implementado withdraw usando a api

--Agora o wihtdraw é feito usando a httpApi e não mais o socket

joaojvds committed:
Pagina de balances pega account pela api

--A lista de account é pega usando a httpApi

GCastilho commented on client/src/routes/balances/index.svelte:
```diff
@@ -15,6 +15,31 @@
+	/**
+	 * retorna uma lista com os detalhes das accounts 
+	 */
+	const fetchCurrenciesList = new Promise(async (resolve, reject) => {
```
https://eslint.org/docs/rules/no-async-promise-executor

joaojvds committed:
Balance agora é pego por http

--A store balance agora é pega usando httpApi
--Consertado exception na hora de realizar a função toFixed()

joaojvds committed:
Merge branch 'master' into implementation-httpApi

GCastilho commented:
O que falta pra implementar aqui?

joaojvds commented:
ficou de você revisar aquele o código para vê se muda ou não aquele promise

GCastilho commented:
Ah, é, verdade
Vou ver como eu resolvo ele

joaojvds commented:
E ele tambem esta dando um erro no test
> 1) Testing operations on the currencyApi

Não sei o porque dele esta dando erro nesse test

GCastilho commented:
Provavelmente pq teve atualização em testes

joaojvds committed:
Merge branch 'master' into implementation-httpApi

GCastilho commented:
Já que vc deu o merge, dá uma olhada e vê se vc consegue corrigir o bug no teste, ok?

GCastilho committed:
Refatorado fetch de currencies

-- Refatorado a função 'fetchCurrenciesList' da balances para não usar um executor assíncrono de promessas

GCastilho committed:
Corrigido teste

-- Corrigindo um teste que estava detectando erroneamente um erro de arredondamento na withdraw

GCastilho reviewed (approved):
Pelo q eu me lembro isso tava certo
Se pegar fogo a gente apaga

GCastilho committed:
Merge branch 'master' into implementation-httpApi
